lib/models/BackendInfo.ts

```diff
@@ -52,7 +52,8 @@ export default class BackendInfo {
         locationConstraint: string | undefined,
         log: RequestLogger,
     ) {
-        if (!locationConstraint || !(locationConstraint in config.locationConstraints)) {
+        if (!locationConstraint || !(locationConstraint in config.locationConstraints) ||
+            config.locationConstraints[locationConstraint].isCrr) {
```

Comment:
```suggestion
            config.locationConstraints[locationConstraint].isCRR) {
```

as this is an acronym?

Reply:
it is defined as `isCrr` in cloudserver and zkop, c.f. other PR, so care must be taken if we change it.